src/runtime-rs/crates/hypervisor/README.md

```diff
@@ -89,6 +90,214 @@ pub trait Hypervisor: Send + Sync {
     async fn save_state(&self) -> Result<HypervisorState>;
    }
 ```
+### VM start procedure: steps through time
 In current design, VM will be started in the following steps.
 
 ![vmm start](../../docs/images/vm-start.svg)
+
+### Semantic of each function of the trait
```

Comment:
Can you put this under the hypervisor trait

Reply:
I think this may fuss the trait. This section is sepatated from the trait for clarity.